src/codegen_internal.h

```diff
@@ -1,6 +1,6 @@
 // This file is a part of Julia. License is MIT: http://julialang.org/license
 
-#if defined(LLVM38) && !defined(LLVM37)
+#if JL_LLVM_VERSION >= 30800
```

Comment:
The function is also declared in `jitlayers.cpp`. Doing it here might have some issue due to missing type declaration/missing headers.